cmake/CMakeLists.txt

```diff
@@ -1,17 +1,28 @@
 cmake_minimum_required(VERSION 3.28)
 project(IsoMaker LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 11)
+
+if(WIN32)
+    set(CMAKE_GENERATOR_TOOLSET "v143")
+    add_compile_options(-Wall)
```

Comment:
we can't -Wextra for windows ?

Reply:
Hey! On windows `-Wall` does `-Wall` and `-Wextra`